lib/serverengine/process_manager.rb

```diff
@@ -101,7 +101,7 @@ def configure(config, opts={})
 
     def fork(&block)
       if ServerEngine.windows?
-        raise NotImplementedError, "fork is not available on this platform. Please use spawn (worker_type: 'spawn')."
+        raise "fork is not available on this platform. Please use spawn (worker_type: 'spawn')."
```

Comment:
RDoc of ruby core says:

> Raised when a feature is not implemented on the current platform. For example, methods depending on the fsync or fork system calls may raise this exception if the underlying operating system or Ruby runtime does not support them.

https://ruby-doc.org/core-2.3.0/NotImplementedError.html

Raising NotImplementedError should be an appropriate behavior.